src/Browscap/Command/CheckDuplicateTestsCommand.php

```diff
@@ -0,0 +1,39 @@
+<?php
+declare(strict_types = 1);
+namespace Browscap\Command;
+
+use Browscap\Helper\IteratorHelper;
+use Browscap\Helper\LoggerHelper;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class CheckDuplicateTestsCommand extends Command
+{
+    protected function configure() : void
+    {
+        $this
+            ->setName('check-duplicate-tests')
+            ->setDescription('checks the test cases for duplicates');
+    }
+
+    /**
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     *
+     * @return int|null null or 0 if everything went fine, or an error code
```

Comment:
Doesn't this only return 1 or 0?

Reply:
This line is copied from the symfony docblock